File: Move Zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        zeros = []
        for idx, n in enumerate(nums):
            if n == 0:
                zeros.append(idx)
                continue
            if(len(zeros) != 0):
                nums[zeros.pop(0)] = n
                nums[idx] = 0
                zeros.append(idx)
                
        

File: leetcode/Move Zeroes.c
void moveZeroes(int* nums, int numsSize){
    signed int idx = -1; // flag
    int i;
    int temp;
    
    for(i=0;i<numsSize;++i){
        if(idx = -1 && nums[i] == 0){
            idx = i;
            continue;
        }
        if(nums[i] == 0)
            continue;
        
        temp = nums[i];
        nums[i] = nums[idx];
        nums[idx] = nums[i];
        
        idx = i;
    }
    
}